sematic/ui/src/types/Types.tsx

```diff
@@ -676,6 +668,7 @@ function DataFrameValueView(props: ValueViewProps) {
 }
 
 function LinkValueView(props: ValueViewProps) {
+  console.log(props)
```

Comment:
Clean-up please 🙂

Reply:
Sorry for that, removed in https://github.com/sematic-ai/sematic/pull/220/commits/13ecac698d3ef97025902e4d8999a2b17a17e489.